src/Altinn.Platform/Altinn.Platform.Authorization/Authorization/Services/Implementation/PolicyAdministrationPoint.cs

```diff
@@ -99,6 +100,219 @@ public async Task<List<Rule>> TryWriteDelegationPolicyRules(List<Rule> rules)
 
             return result;
         }
+                
+        private async Task<List<Rule>> ProcessPolicyFile(string policyPath, string org, string app, RequestToDelete deleteRequest)
+        {
+            List<Rule> currentRules = new List<Rule>();
+            string leaseId = await _policyRepository.TryAcquireBlobLease(policyPath);
+            if (leaseId == null)
+            {
+                _logger.LogInformation($"Could not acquire blob lease lock on delegation policy at path: {policyPath}");
```

Comment:
For rikere logging bør alle logg kall gjøres som:
_logger.LogInformation("Could not acquire blob lease lock on delegation policy at path: {policyPath}", policyPath);

Reply:
Fixed

Reply:
Fixed for all I found